internal/provider/datasource_cloud_connections.go

```diff
@@ -239,6 +239,14 @@ func dataSourceCloudConnections() *schema.Resource {
 										Type:     schema.TypeString,
 										Computed: true,
 									},
+									"primary_public_ip": {
```

Comment:
@vgvm-pf you forgot to run the `tfplugindocs.sh` script as you added new attributes to the schema in the data source. Those won't show on the doc registry.

Reply:
Oh... you're right...

Reply:
Yeah... you're right. The field fix was a bit last minute... new PR for it: https://github.com/PacketFabric/terraform-provider-packetfabric/pull/537